L/lexbor/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "lexbor"
+version = v"2.3.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/lexbor/lexbor.git", "26a8819b60801301df92d771f12a01dbe8705a7a")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/lexbor
+cmake -B build -G "Unix Makefiles" -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DLEXBOR_BUILD_SEPARATELY=ON -DLEXBOR_BUILD_STATIC=OFF
```

Comment:
`-G "Unix Makefile"` comes from their docs for building on Windows: https://lexbor.com/docs/lexbor/#windows. Wasn't required for other platforms but seems to build fine with it added. Could always do it conditionally for Windows only if required.

`-DLEXBOR_BUILD_SEPARATELY=ON` added to get separate libs for each module that they provide.

`-DLEXBOR_BUILD_STATIC=OFF` added since I figured we don't really need those, the dynlibs only is fine?

Reply:
I think unix makefiles is the default anyway for us.